package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "rollup-plugin-css-modules",
```

Comment:
it looks like someone's squatting on this package name
https://www.npmjs.com/package/rollup-plugin-css-modules

in the readme you have it scoped to your username so it should be the same here.

but also maybe worth considering calling this `rollup-plugin-css-module-scripts` to disambiguate from the other thing?